app/commands/exercise/update_important_files_hash_with_solutions.rb

```diff
@@ -0,0 +1,28 @@
+class Exercise
+  class UpdateImportantFilesHashWithSolutions
```

Comment:
I think we should rename this to reflect the fact it's only called in exceptional circumstances where the algorithm for editor files has changed.

Maybe `RecalculateImportantFilesHash` and then add a comment at the top saying this is intended to be run only as a manual job etc?

Reply:
I've renamed it to `RecalculateImportantFilesHashForSolutions` to stress that it is the solutions' important files hash is being recalculated.

Reply:
See https://github.com/exercism/website/pull/2049/commits/56eee271010c1e0af20e2e0be9a54c06d10ed4ef